3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java

```diff
@@ -334,60 +342,81 @@ private void initialize() {
         m_routingOptionType.setSelection(0);
         m_routingOptionMode.setSelection(1);
 
-        // initialise the Map Fragment to have a map created and attached to
-        // the fragment
-        m_mapFragment.init(new OnEngineInitListener() {
-            @Override
-            public void onEngineInitializationCompleted(Error error) {
-                if (error == Error.NONE) {
-                    // retrieve a reference of the map from the map fragment
-                    m_map = m_mapFragment.getMap();
-                    // Set the map center, zoom level, orientation and tilt
-                    m_map.setCenter(new GeoCoordinate(49.196261, -123.004773, 0.0),
-                            Map.Animation.NONE);
-                } else {
-                    System.out.println("ERROR: Cannot initialize Map Fragment" + error.toString());
+        // Set path of isolated disk cache
+        String diskCacheRoot = Environment.getExternalStorageDirectory().getPath()
+                + File.separator + ".isolated-here-maps";
+        // Retrieve intent name from manifest
+        String intentName = "";
+        try {
+            ApplicationInfo ai = getPackageManager().getApplicationInfo(m_activity.getPackageName(), PackageManager.GET_META_DATA);
+            Bundle bundle = ai.metaData;
+            intentName= bundle.getString("INTENT_NAME");
```

Comment:
Fix up spacing

Reply:
Yes, just noticed after pushed them in. Working on the formatting now...